sklearn/feature_extraction/text.py

```diff
@@ -1440,6 +1440,28 @@ class TfidfVectorizer(CountVectorizer):
     sublinear_tf : boolean, default=False
         Apply sublinear tf scaling, i.e. replace tf with 1 + log(tf).
 
+    CountVectorizer : sparse matrix, (n_samples, n_features)
```

Comment:
Putting these here would suggest that `CountVectorizer` is an `__init__` parameter to the class, which it isn't. That's why travis is failing.

These information you have here are useful (IMO), but I guess the idea here is to also explain what these classes do by better explaining the parameters already listed in the docstring. For instance, the `norm` parameter was mentioned as badly documented in the issue. As an example, you can have a look at the parameter descriptions of [`LogisticRegression`](https://scikit-learn.org/dev/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression), I hope it helps.

Reply:
@adrinjalali 
I updated some descriptions.  Is this better?